src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -48,11 +48,13 @@ internal virtual bool RequiresCompletion
             get { return false; }
         }
 
-        internal virtual void ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
+#nullable enable
+        internal virtual void ForceComplete(SourceLocation? locationOpt, Predicate<ISymbolInternal>? filter, CancellationToken cancellationToken)
```

Comment:
>Predicate<ISymbolInternal> [](http://example.com/codeflow?start=73&length=26)

I think we should switch this type to ```Predicate<Symbol>``` for this API and for other similar APIs as soon as all callers are inside C# compiler.

Reply:
For example this is done for `MethodCompiler.CompileMethodBodies` today.